### ISSUE TITLE
Modify LOOKUP family to return text cells?

### ISSUE THREAD

**Issue by cesss** [andmarti1424/sc-im#478]
Is it possible to patch the LOOKUP/HLOOKUP/VLOOKUP family of functions so that they return text if the result is a text cell? Currently they return 0 in that case, but however one of the most useful applications of these functions is to return text (returning a label or message depending on the value on other row or column).

I tried to modify it myself but couldn’t find a way: can a function return either a number or text? Is there some workaround that I can use? It would be very useful for me!

**Comment by andmarti1424** [comment]
No. You can't. In C you have to return a struct with those two values,and one would be null or undefined value..
